Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1846,14 +1847,22 @@ extension BrowserViewController: IntroViewControllerDelegate {
     }
 
     func presentSignInViewController() {
-        // TODO When bug 1161151 has been resolved we can jump directly to the sign in screen
-        let settingsNavigationController = SettingsNavigationController()
-        settingsNavigationController.profile = self.profile
-        settingsNavigationController.tabManager = self.tabManager
+        let signInVC = FxAContentViewController()
+        signInVC.delegate = self
```

Comment:
I noticed `FxAContentViewController` `delegate` is not `weak`. Can we fix that while here?